nixos/tests/web-apps/peertube.nix

```diff
@@ -152,14 +176,46 @@ import ../make-test-python.nix (
       server.wait_for_open_port(9000)
 
       # Check if PeerTube is running
-      client.succeed("curl --fail http://peertube.local:9000/api/v1/config/about | jq -r '.instance.name' | grep 'PeerTube\ Test\ Server'")
+      client.succeed("curl --fail http://peertube.local:9000/api/v1/config/about | jq -r '.instance.name' | grep 'PeerTube Test Server'")
+
+
+      # PeerTube CLI
 
-      # Check PeerTube CLI version
       client.succeed('peertube-cli auth add -u "http://peertube.local:9000" -U "root" --password "zw4SqYVdcsXUfRX8aaFX"')
       client.succeed('peertube-cli auth list | grep "http://peertube.local:9000"')
       client.succeed('peertube-cli auth del "http://peertube.local:9000"')
       client.fail('peertube-cli auth list | grep "http://peertube.local:9000"')
 
+
+      # peertube-runner
+
+      access_token = client.succeed(
+        'peertube-cli get-access-token --url "http://peertube.local:9000" --username "root" --password "zw4SqYVdcsXUfRX8aaFX"'
+      ).strip()
+      # Generate registration token.
+      client.succeed(f"curl --fail -X POST -H 'Authorization: Bearer {access_token}' http://peertube.local:9000/api/v1/runners/registration-tokens/generate")
+      # Get registration token, and put it where `registrationTokenFile` from the
+      # peertube-runner module points to.
+      client.succeed(
+        f"curl --fail -H 'Authorization: Bearer {access_token}' http://peertube.local:9000/api/v1/runners/registration-tokens" \
+        " | jq --raw-output '.data[0].registrationToken'" \
+        " > /etc/peertube-runner-registration-token"
```

Comment:
Maybe put the expected path into a `let in` block, *juuuuust* to make sure that they're always changed together?

Reply:
I was following the precedent in this file where stuff like the password and URL also  aren't defined. I can change them all if you think that would be fine.

Reply:
Personally I would prefer that, because I would expect all these to be changed together via substitution into the config & script instead of copy-pasting. But I wouldn't block this PR on that. Feel free to mark this comment as resolved and either leave it as-is or implement the suggestion.